pkg/api/types.go

```diff
@@ -1357,6 +1357,20 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty"`
 }
 
+// DaemonEndpoint store information about a single Daemon endpoint
+type DaemonEndpoint struct {
+	// Port number of the given endpoint
+	Port int `json:port`
+}
+
+// NodeDaemonEndpoints lists ports opened by daemons running on the Node
+type NodeDaemonEndpoints struct {
+	// Endpoint on which Kubelet is listening
+	KubeletEndpoint DaemonEndpoint `json:"kubeletEndpoint,omitempty"`
+	// Endpoint opened by Kubelet on which cAdvisor proxy is available
```

Comment:
That I don't know. We didn't want to pollute Kubelet endpoint? @dchen1107 @yujuhong